### PR TITLE
Issue 2560 - Safe View/Settings Updates

### LINKED ISSUE #2560: Safe View/Settings
https://www.notion.so/hulking-cosmos-3e8/Q4-2024-QA-Requirements-13e2ee9c5bc280a88458e7a935d17488?pvs=4#1422ee9c5bc28078bb8cdf6de157dd1f

### PR DESCRIPTION
**Pull request by adamgall**
Closes #2560

Single propose button, updated copy, conditional "snapshot" copy.

when a snapshot space is not already set:
Screenshot 2024-11-21 at 11 27 52 AM

when a snapshot space is already set:
Screenshot 2024-11-21 at 11 29 06 AM

### CODE REVIEW

**Review by parkermccurley — COMMENTED** [review]
Thanks Adam looks great

File: src/i18n/locales/en/common.json [hunk 1]
```diff
@@ -117,5 +117,6 @@
   "ipfsLoadingErrorMessage": "Error fetching data from IPFS, please try again later.",
   "ipfsSavingErrorMessage": "Error saving data to IPFS, please try again later.",
   "confirm": "Confirm",
-  "automaticChainSwitchingErrorMessage": "We couldn't automatically switch network to the Safe's network. Please, try to switch network in your wallet application."
+  "automaticChainSwitchingErrorMessage": "We couldn't automatically switch network to the Safe's network. Please, try to switch network in your wallet application.",
```
Comment (DarksightKellar) [inline src/i18n/locales/en/common.json:120]: Doesn't look like this diff is an actuall part of the PR, but I also don't think this notification copy was caught in the QA. Reference to "Safe" here

**Review by DarksightKellar — APPROVED** [review]
lgtm

File: src/i18n/locales/en/common.json (hunk 1 above)
Comment (adamgall) [reply src/i18n/locales/en/common.json]: pushed

**Review by Da-Colon — APPROVED** [review]

**Pull request merged**